src/overlays/actors/ovl_En_Kbt/z_en_kbt.c

```diff
@@ -31,36 +34,543 @@ const ActorInit En_Kbt_InitVars = {
+
+        case 0xC3A:
+        case 0xC4D:
+        case 0xC58:
+            this->unk_27E = 0;
+            this->unk_27F = 1;
+            this->unk_282 = 8;
+            break;
+
+        case 0xC3C:
+        case 0xC57:
+            this->unk_27E = 1;
+            this->unk_282 = 7;
+            break;
+
+        case 0xC3D:
+            this->unk_27E = 0;
+            break;
+
+        case 0xC3E:
+            this->unk_27F = 1;
+            this->unk_282 = 8;
+            break;
+
+        case 0xC3F:
+        case 0xC47:
+        case 0xC4C:
+            this->unk_27E = 1;
+            this->unk_27F = 1;
+            this->unk_282 = 7;
+            break;
+
+        case 0xC40:
+            this->unk_27E = 3;
+            this->unk_282 = 6;
+            break;
+
+        case 0xC45:
+        case 0xC4B:
+        case 0xC56:
+            this->unk_27E = 0;
+            this->unk_27F = 1;
+            this->unk_282 = 7;
+            break;
+
+        case 0xC46:
+            this->unk_27E = 1;
+            this->unk_27F = 1;
+            this->unk_282 = 9;
+            break;
+
+        case 0xC48:
+            this->unk_27E = 2;
+            this->unk_27F = 0;
+            this->unk_282 = 10;
+            break;
+
+        case 0xC49:
+            this->unk_27F = 1;
+            break;
+
+        case 0xC4A:
+            this->unk_27E = 0;
+            this->unk_282 = 7;
+            break;
+
+        case 0xC4E:
+            this->unk_27E = 1;
+            this->unk_27F = 1;
+            this->unk_282 = 12;
+            break;
+
+        case 0xC4F:
+            this->unk_27E = 2;
+            this->unk_27F = 1;
+            this->unk_282 = 12;
+            break;
+
+        case 0xC50:
+            this->unk_27E = 0;
+            this->unk_27F = 1;
+            this->unk_282 = 12;
+            break;
+    }
+
+    this->actor.textId = 0;
+    func_80B3415C(this);
+}
+
+void EnKbt_Update(Actor* thisx, GlobalContext* globalCtx) {
+    EnKbt* this = THIS;
+
+    this->actionFunc(this, globalCtx);
+}
+
+s32 EnKbt_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx) {
+    EnKbt* this = THIS;
+
+    if (!(this->unk_27C & 1) && (limbIndex == OBJECT_KBT_LIMB_0E)) {
+        *dList = NULL;
+    }
+    return false;
+}
+
+Vec3f D_80B34B84 = { 500.0f, 500.0f, 0.0f };
+
+void EnKbt_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx) {
+    EnKbt* this = THIS;
+
+    if (limbIndex == OBJECT_KBT_LIMB_09) {
+        Matrix_MultiplyVector3fByState(&D_80B34B84, &this->actor.focus.pos);
+    }
+}
+
+TexturePtr D_80B34B90[] = {
+    object_kbt_Tex_00A5B8,
+    object_kbt_Tex_00ADB8,
+};
+
+TexturePtr D_80B34B98[] = {
+    object_kbt_Tex_00B5B8, object_kbt_Tex_00BDB8, object_kbt_Tex_00C5B8, object_kbt_Tex_00CDB8, object_kbt_Tex_00D5B8,
+};
+
+void EnKbt_Draw(Actor* thisx, GlobalContext* globalCtx) {
+    EnKbt* this = THIS;
+    Gfx* gfx;
+    void* temp;
```

Comment:
```suggestion
    TexutrePtr tex;
```
Used as a ptr to a texture